message TioMessage {
  optional string request_id = 1;
  optional string full_method_name = 2;
  optional string response_container_name = 3;
 
 enum Type {
    REQUEST = 0;
    RESPONSE = 1;
    PUBLICATION = 2;
  }
  
  optional Type type = 4 [default = REQUEST];
  
  optional bytes payload = 5;
}
